dependencies/src/main/java/com/google/cloud/tools/opensource/classpath/ClassPathBuilder.java

```diff
@@ -74,7 +77,7 @@
    * @return an ordered map of absolute paths of jar files to one or more Maven dependency paths
    * @throws RepositoryException when there is a problem retrieving jar files
    */
-  public static LinkedListMultimap<Path, DependencyPath> artifactsToDependencyPaths(
+  public LinkedListMultimap<Path, DependencyPath> artifactsToDependencyPaths(
```

Comment:
why would this method need to change?

Reply:
The `levelOrder` method needs to be non-static to touch non-static `artifactProblems`. Then all other methods that may call levelOrder becomes non-static.